tests_e2e/test_suites/ext_cgroups.yml

```diff
@@ -0,0 +1,9 @@
+#
+# The test suite installs the few extensions and
+# verify those extensions are running in expected cgroups and also, checks agent tracking those cgroups for polling resource metrics.
+#
+name: "ExtCgroups"
+tests:
+  - "ext_cgroups/ext_cgroups.py"
+images: "cgroups-endorsed"
+locations: "AzureCloud:southcentralus"
```

Comment:
is this restricted to south central us because the GATestExt is only published there? If so, may be helpful to have a comment here

Reply:
but I would do it in next release as this required to create pipeline to deploy.

Reply:
Created work item [Task 24842472](https://msazure.visualstudio.com/One/_workitems/edit/24842472): Release GATest extension to other regions

Reply:
why do you need this specific extension in this test?

Reply:
I want to test services cgroups which gets installed by extensions. Our extension install sample service

Reply:
ok, please add a comment

Reply:
(and same if you are using any specific extension for a specific purpose)

Reply:
Sorry, I meant adding a comment about why we are using that extension.